src/control/cmd/dmg/server.go

```diff
@@ -0,0 +1,58 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/lib/control"
+)
+
+// ServerCmd is the struct representing the top-level server subcommand.
+type ServerCmd struct {
```

Comment:
Think we have some crossed wires, you requested that the fanout happens client side and we discussed that in that case it would be a hostlist base command and not MS. You also requested that it be "dmg server set-loglevel" so I've implemented it as such (except I have called it set-logmasks to be consistent with dlog with which it interacts).